src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj

```diff
@@ -10,7 +10,7 @@
     <ProjectGuid>{F75E3008-0562-42DF-BE72-C1384F12157E}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AssemblyName>System.Net.Http.WinHttpHandler</AssemblyName>
-    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <AssemblyVersion>4.0.0.1</AssemblyVersion>
```

Comment:
>This permits folks to install the assembly in the GAC 

We allow customers to put .NET Core binaries into the GAC??  I thought the GAC was for Desktop (.NET Framework) only.